gatsby/content/blog/2023/02/2023-02-16-synapse-1.77.mdx

```diff
@@ -0,0 +1,46 @@
+---
+date: '2023-02-16'
+title: Synapse 1.77 released
+categories:
+- Releases
+author: Mathieu Velten
+---
+
+Greetings Matrix fans! We've published Synapse [version 1.77](https://github.com/matrix-org/synapse/releases/tag/v1.77.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience.
+
+## The new stuff
+
+### Experimental support for intentional mentions (MSC3952)
+
+Mentioning other users on Matrix is difficult: it is not possible to know if mentioning a user
+by display name or Matrix ID will count as a mention, but is also too easy to mistakenly mention a user.
+
+[MSC3952](https://github.com/matrix-org/matrix-spec-proposals/pull/3952) is a way to solve that trouble
+by making the mentioning explicit in the protocol using a dedicated event property,
+instead of relying on searching the body of the message as before.
```

Comment:
```suggestion
[MSC3952](https://github.com/matrix-org/matrix-spec-proposals/pull/3952) proposes a solution. Its idea is to make the mentioning explicit in the protocol using a dedicated event property,
instead of relying on searching the body of the message as before.
```